src/DynamoRevit/DynamoRevit.cs

```diff
@@ -367,8 +361,27 @@ public static string GetGeometryFactoryPath(string corePath)
                 new object[] { corePath, Versions.ShapeManager }) as string);
         }
 
+        private static void PreloadDynamoCoreDlls()
+        {
+            // Assume Revit Install folder as look for root. Assembly name is compromised.
+            var assemblyList = new[]
+            {
+                "SDA\\bin\\ICSharpCode.AvalonEdit.dll"
+            };
+
+            foreach (var assembly in assemblyList)
+            {
+                var assemblyPath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory + "", assembly);
+                if(File.Exists(assemblyPath))
+                    Assembly.LoadFrom(assemblyPath);
+            }
+        }
+
         private static RevitDynamoModel InitializeCoreModel(DynamoRevitCommandData commandData)
         {
+            // Temporary fix to pre-load DLLs that were also referenced in Revit folder. 
+            // To do: Need to align with Revit when provided a chance.
+            PreloadDynamoCoreDlls();
```

Comment:
@QilongTang from your analysis did you ever see a assembly resolve event or exception thrown when we load the second dll (whichever it may be)?